src/plugins/intel_gna/frontend/quantized_layer_params.hpp

```diff
@@ -66,6 +66,15 @@ class Quantization {
         SetMinValues(src.GetMinValues(false), false);
         SetMaxValues(src.GetMaxValues(false), false);
     }
+    void Reset() {
+        scale = 1.0f;
```

Comment:
I would prefer to use constants where it is possible: scale = kScaleFactorDefault?

Reply:
Sure will do